recipes/botan/all/test_package/conanfile.py

```diff
@@ -0,0 +1,24 @@
+import os
+from conans import ConanFile, CMake
+
+
+class TestPackageConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake"
+
+    def configure(self):
```

Comment:
Remove it, do not force a setting. If something is not supported, raise an ConanInvalidConfiguration from package recipe.

Reply:
Yes, I like that. I wasn't aware that the build-bot skips configurations that result in a `ConanInvalidConfiguration` exception.